docs(README): add note on features not supported by Chromium

ithinkihaveacat opened the pull request:
Developers are frequently surprised that Puppeteer does not support video (see e.g. https://github.com/GoogleChrome/puppeteer/issues/291 and [other issues mentioning "video"](https://github.com/GoogleChrome/puppeteer/issues?utf8=%E2%9C%93&q=is%3Aissue+video), many of which are related).

The documentation does explain how to use Chrome instead of Chromium, but I couldn't find anywhere that links this "fix" to the problem of audio/video not working (the only reason to do this?).

ithinkihaveacat committed:
Add features not supported by Puppeteer

ebidel commented on README.md:
```diff
@@ -219,3 +219,10 @@ other frameworks could adopt Puppeteer as their foundational layer.
+
+You may find that Puppeteer does not behave as expected when controlling pages that incorporate audio and video. (For example, [video playback/screenshots is likely to fail](https://github.com/GoogleChrome/puppeteer/issues/291).) There are two reasons for this:
+
+* Puppeteer is bundled with Chromium--not Chrome--and so by default, it inherits all of [Chromium's video-related limitations](https://www.chromium.org/audio-video). This means that Puppeteer does not understand the MP4 container format, and also does not support MP3, AAC or H.264. (However, it is possible to force Puppeteer to use a separately-installed version Chrome instead of Chromium via the ...
```
Thanks for adding this. mp3/mp4 support was added to chromium 2 days ago. https://bugs.chromium.org/p/chromium/issues/detail?id=746579

We should probably leave them out since we'll pick them up the next time we rev chromium.

ithinkihaveacat replied:
Sure, that's good news! Updated to remove mention of MP3/MP4.

ithinkihaveacat committed:
Update README.md

Remove mention of MP3/MP4, since they will be supported by Chromium within days.

ebidel commented on README.md:
```diff
@@ -219,3 +219,10 @@ other frameworks could adopt Puppeteer as their foundational layer.
+
+#### Q: What features does Puppeteer not support?
+
+You may find that Puppeteer does not behave as expected when controlling pages that incorporate audio and video. (For example, [video playback/screenshots is likely to fail](https://github.com/GoogleChrome/puppeteer/issues/291).) There are two reasons for this:
```
Sorry to be picky. I think we can ditch the link now b/c it's discussing lack of mp3/4 support, which is not true anymore :)

ithinkihaveacat replied:
Is MP4 supposed to work now with GoogleChrome/master? #291 now fails in a slightly different way (no error message), but the video still doesn't play.

aslushnikov replied:
Indeed, looks like mp4 is still not supported by chromium atm.

ebidel commented on README.md:
```diff
@@ -219,3 +219,10 @@ other frameworks could adopt Puppeteer as their foundational layer.
+
+You may find that Puppeteer does not behave as expected when controlling pages that incorporate audio and video. (For example, [video playback/screenshots is likely to fail](https://github.com/GoogleChrome/puppeteer/issues/291).) There are two reasons for this:
+
+* Puppeteer is bundled with Chromium--not Chrome--and so by default, it inherits all of [Chromium's media-related limitations](https://www.chromium.org/audio-video). This means that Puppeteer does not support AAC or H.264. (However, it is possible to force Puppeteer to use a separately-installed version Chrome instead of Chromium via the [`executablePath` option to `puppeteer.launch`](https://git ...
```
Puppeteer does not support AAC or H.264 -> Puppeteer does not support licensed formats such as AAC or H.264.

ithinkihaveacat replied:
Changed in 5b0d01878eab40d59ff252a837195087322b9ed1.

ebidel commented on README.md:
```diff
@@ -219,3 +219,10 @@ other frameworks could adopt Puppeteer as their foundational layer.
+
+You may find that Puppeteer does not behave as expected when controlling pages that incorporate audio and video. (For example, [video playback/screenshots is likely to fail](https://github.com/GoogleChrome/puppeteer/issues/291).) There are two reasons for this:
+
+* Puppeteer is bundled with Chromium--not Chrome--and so by default, it inherits all of [Chromium's media-related limitations](https://www.chromium.org/audio-video). This means that Puppeteer does not support AAC or H.264. (However, it is possible to force Puppeteer to use a separately-installed version Chrome instead of Chromium via the [`executablePath` option to `puppeteer.launch`](https://git ...
```
if you need Chrome's audio/video support. -> if you need an official release of Chrome that supports these media formats.

ithinkihaveacat replied:
Changed in b4ee320b517d1fee3baba3bbeeb4003a81eb5c36.

aslushnikov commented:
@ithinkihaveacat do you want to proceed with this?

ithinkihaveacat commented:
@aslushnikov Yes, sorry, I just just waiting to see what forms of video Chromium/Puppeteer supports with the latest update, but I'm not sure there's any change--is there supposed to be?

aslushnikov commented:
@ithinkihaveacat I don't think anything has changed.

ithinkihaveacat committed:
Update README.md

ithinkihaveacat committed:
Update README.md

ithinkihaveacat commented:
@aslushnikov Ok, I think this is ready now.

@ebidel wanted the bit that mentions video playback being likely to fail removed, but in my tests I don't think Puppeteer's behaviour has changed much here, so I've left it in.
